tools/jira-release.ts

```diff
@@ -0,0 +1,138 @@
+import { program } from 'commander'
```

Comment:
Converted to typescript and converted manual oauth get/post/put with jira api calls.